source/Public/Utils/Get-FabricLongRunningOperation.ps1

```diff
@@ -58,13 +58,9 @@ This command polls the status of the operation with the given operationId every
             }
 
             # Step 3: Make the API request
-            $response = Invoke-RestMethod `
-                -Headers $FabricConfig.FabricHeaders `
+            $response = Invoke-FabricRestMethod `
```

Comment:
No Test-TokenExpired before this on this one?

Reply:
No, that's the original version of the code, which I did not intend to change as it's outside the scope of this PR.